docs/modules/ROOT/pages/pallets/pallet_xcm.adoc

```diff
@@ -0,0 +1,340 @@
+** `TrustedLockers` -- The assets which we consider a given origin is trusted if they claim to have placed a lock. Type must implement `ContainsPair`.
+
+* Pallet-specific converters:
+** `CurrencyMatcher` -- The `MultiAsset` matcher for `Currency`. Type must implement the trait `MatchesFungible`.
+** `SovereignAccountOf` -- How to get an `AccountId` value from a `MultiLocation`, useful for handling asset locks. Type must implement `ConvertLocation`.
+
+* Pallet-specific constants:
+** `VERSION_DISCOVERY_QUEUE_SIZE` -- `u32` measures the size of the version discovery queue. Typically to set to `100`.
+** `AdvertisedXcmVersion` -- The latest supported XCM version that this chain accepts. Type must implement the trait `Get<XcmVersion>`. Commonly set to `pallet_xcm::CurrentXcmVersion`.
+** `MaxLockers` --  The maximum number of local XCM locks that a single account may have. Type must implement the trait `Get<u32>`.
+** `MaxRemoteLockConsumers` -- The maximum number of consumers a single remote lock may have. Type must implement the trait `Get<u32>`.
+** `UniversalLocation` -- This chain's Universal Location. Type must implement the trait `Get<InteriorMultiLocation>`.
+
+* Benchmark-only configs:
+** `ReachableDest` -- A `MultiLocation` that can be reached via `XcmRouter`. Used only in benchmarks. Type must implement trait `ReachableDest`.
+
+* Common configs:
+** `RuntimeEvent`
+** `RuntimeCall`
+** `RuntimeOrigin`
+** `WeightInfo`
+
+== Dispatchables
+
+[.contract-item]
+[[send]]
+==== `[.contract-item-name]#++send++#`
+[source,rust]
+----
+pub fn send(
+    origin: OriginFor<T>,
+    dest: Box<VersionedMultiLocation>,
+    message: Box<VersionedXcm<()>>,
+)
+----
+Send a versioned XCM `message` to the destination `dest`.
+
+The origin must be `SendXcmOrigin` for this call.
+
+**Params:**
+
+- `dest: Box<VersionedMultiLocation>` — destination for the XCM
+- `message: Box<VersionedXcm<()>>` — versioned XCM to be sent to the multilocation `dest`
+
+**Errors:**
+
+- `InvalidOrigin` — origin did not match `SendXcmOrigin`
+- `BadVersion` — version for XCM not valid
+
+**Events:**
+
+- `Sent(origin, destination, message, message_id)` -- The versioned XCM `message` was sent from the `origin` to the `destination`.
+
+
+[.contract-item]
+[[execute]]
+==== `[.contract-item-name]#++execute++#`
+[source,rust]
+----
+pub fn execute(
+    origin: OriginFor<T>,
+    message: Box<VersionedXcm<<T as Config>::RuntimeCall>>,
+    max_weight: Weight,
+)
+----
+Execute an XCM message from a local, signed, origin.
+
+The origin must be `ExecuteXcmOrigin` for this call.
+
+NOTE: A successful return to this does NOT imply that the `msg` was executed successfully to completion; only that SOME of it was executed.
+
+**Params:**
+
+- `message: Box<VersionedXcm<<T as Config>::RuntimeCall>>` — versioned XCM to be executed
```

Comment:
There is some weird glitch in compiled version, it is rendered as a link.
Could you please try to fix it?